Controllers/AtualizacaoController.cs

```diff
@@ -0,0 +1,130 @@
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using PRIORI_SERVICES_API.Model;
+using PRIORI_SERVICES_API.Models;
+using PRIORI_SERVICES_API.Models.Dbos;
+using System.Data;
+
+namespace PRIORI_SERVICES_API.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class AtualizacaoController : ControllerBase
+    {
+        private readonly PrioriDbContext _context;
+        public AtualizacaoController(PrioriDbContext context)
+        {
+            _context = context;
+        }
+
+        [HttpGet(Name = "GetAtualizacao")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
+        public async Task<ActionResult<IEnumerable<Atualizacao>>> GetAtualizacao()
+        {
+            return await _context.tblAtualizacao.ToListAsync();
+        }
+
+        [HttpGet("{id}", Name = "GetAtualizacaoById")]
+        [ProducesResponseType(StatusCodes.Status200OK)]
+        [ProducesResponseType(StatusCodes.Status404NotFound)]
+        public async Task<ActionResult<IEnumerable<Atualizacao>>> GetCategoriaFromID(int id)
```

Comment:
Você não atualizou o nome aqui
```suggestion
        public async Task<ActionResult<IEnumerable<Atualizacao>>> GetAtualizacaoByID(int id)
```